### PR TITLE
added mongo db connection url and batch name delete

### PR DESCRIPTION
**Pull request by Olusheenor**
added mongo db connection url and batch name delete endpoint

### CODE REVIEW

File: Api/Controllers/NamesController.cs [hunk 1]
```diff
@@ -164,5 +165,29 @@ public async Task<IActionResult> DeleteName(string name)
             await _nameEntryService.Delete(name);
             return Ok(new { Message = $"{name} Deleted" });
         }
+
+
+        [HttpDelete("batch")]
+        public async Task<IActionResult> DeleteNamesBatch(string[]names)
+        {
+
+            var foundNames = (await _nameEntryService.LoadNames(names))?.Select(f=>f.Name)?.ToArray();
```
Comment (Zifah) [inline Api/Controllers/NamesController.cs:174]: Not sure, but I think this method would either return an empty array (not a null) if no names are found. Please test and modify the conditional statement below if necessary.

Comment (Olusheenor) [reply Api/Controllers/NamesController.cs:174]: Handled empty array.

**Review by Zifah — APPROVED** [review]

File: Application/Cache/NameDeletedCache.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿using Core.Cache;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Application.Cache
+{
+    public class NameDeletedCache :InMemoryCache, INameDeletedCache
```
Comment (Zifah) [inline Application/Cache/NameDeletedCache.cs:10]: You can safely remove this cache. We won't be needing it.

Comment (Olusheenor) [reply Application/Cache/NameDeletedCache.cs]: Done

File: Core/Dto/Response/IndexOperationStatusDto.cs
Comment (Zifah) [inline Core/Dto/Response/IndexOperationStatusDto.cs:1]: You can remove this file as well. We're not using it anymore. Feel free to merge in after that.

**Review by Zifah — APPROVED** [review]

**Review by Zifah — APPROVED** [review]

**Pull request merged**